### DISCUSSION TITLE
google-services.json not found, google-services plugin not applied. Push Notifications won't work

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [ionic-team/capacitor#3332] [Q&A] [answered]
Hi,

Can you tell me why the below error shows to me on the Android Studio project? I have put the `google-services.json` file under the `android/app` folder.  My app doesn't use Firebase Push. But it has a Firebase Phone Auth.

![c1](https://user-images.githubusercontent.com/5206526/88464571-843c7080-ced9-11ea-9dd3-77159547629a.png)

![c2](https://user-images.githubusercontent.com/5206526/88464583-a8984d00-ced9-11ea-879b-e9b4069a7829.png)

_package.json_

{
  "name": "hello",
  "version": "2.0.0",
  "author": "Hello",
  "homepage": "home.com/",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "9.1.6",
    "@angular/core": "9.1.6",
    "@angular/fire": "6.0.0",
    "@angular/forms": "9.1.6",
    "@angular/platform-browser": "9.1.6",
    "@angular/platform-browser-dynamic": "9.1.6",
    "@angular/router": "9.1.6",
    "@capacitor/android": "^2.1.0",
    "@capacitor/core": "2.2.0",
    "@capacitor/ios": "^2.2.0",
    "@ionic-native/core": "5.26.0",
    "@ionic-native/firebase-authentication": "^5.26.0",
    "@ionic-native/in-app-browser": "^5.26.0",
    "@ionic-native/screen-orientation": "^5.26.0",
    "@ionic/angular": "5.2.3",
    "@ngx-translate/core": "^12.1.2",
    "@ngx-translate/http-loader": "^4.0.0",
    "angular2-signaturepad": "^2.11.0",
    "cordova-plugin-firebase-authentication": "^3.3.0",
    "cordova-plugin-inappbrowser": "^4.0.0",
    "cordova-plugin-screen-orientation": "^3.0.2",
    "cordova-support-android-plugin": "^1.0.2",
    "cordova-support-google-services": "^1.4.0",
    "dayjs": "^1.8.27",
    "es6-promise-plugin": "^4.2.2",
    "firebase": "7.15.5",
    "rxjs": "6.5.1",
    "signature_pad": "^3.0.0-beta.3",
    "tslib": "1.10.0",
    "zone.js": "0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/architect": "<0.900 || ^0.900.0-0 || ^9.0.0-0",
    "@angular-devkit/build-angular": "^0.901.7",
    "@angular/cli": "9.1.5",
    "@angular/compiler": "9.1.6",
    "@angular/compiler-cli": "9.1.6",
    "@angular/language-service": "9.1.6",
    "@capacitor/cli": "2.1.0",
    "@ionic/angular-toolkit": "2.1.1",
    "@types/node": "12.11.1",
    "codelyzer": "5.1.2",
    "firebase-tools": "8.4.2",
    "fuzzy": "^0.1.3",
    "inquirer": "^6.2.2",
    "inquirer-autocomplete-prompt": "^1.0.1",
    "ts-node": "8.3.0",
    "tslint": "6.1.0",
    "typescript": "3.8.3"
  },
  "description": "hello"
}

**Comment by musps** [comment]
If your file is correctly inside `/android/app/google-services.json`

Try running these actions from android studio:

Edit => Sync project with Gradle files
Edit => Invalidate Caches / Restart...

**Reply by khurramIlyas** [reply]
Now, these option are in File menu, not in Edit

**Reply by alexandermorgan** [reply]
> If your file is correctly inside `/android/app/google-services.json`
> 
> Try running these actions from android studio:
> 
> Edit => Sync project with Gradle files
> Edit => Invalidate Caches / Restart...

@musps , does anything have to be in the google-services.json? I don't have the file but I can put an empty .json file in the directory you specified.

**Comment by vigamage** [comment]
Did you solve the issue?

**Reply by Sampath-Lokuge** [reply]
Yes, just delete it `google-services.json` and add it again through Android Studio.

**Reply by nishara22** [reply]
How do you add it through Android Studio?

**Comment by Sampath-Lokuge** [comment, marked as answer]
Yes, just delete it `google-services.json` and add it again through Android Studio.

**Reply by MaldiniBeh** [reply]
Hi what's up? I've got same issue but I don't know how to fix it.
Can you give me the path of  Android Studio.?

**Reply by MaldiniBeh** [reply]
I'm looking for where to put the file in Anroid studion but I can't do it

**Reply by AruljothySundaramoorthy** [reply]
This too not working, i have tooks a build using ionic with capacitor
![Screenshot_2](https://github.com/ionic-team/capacitor/assets/28552464/3dde7be2-3ec0-459e-ab1a-f425c45131c6)
